llvm/test/CodeGen/AArch64/arm64-vadd.ll

```diff
@@ -1193,6 +1193,366 @@ define <2 x i64> @ssubl2_duplhs(i32 %lhs, <4 x i32> %rhs) {
   ret <2 x i64> %res
 }
 
+define <8 x i32> @saddl_v8i8_v8i32(<8 x i8> %a, <8 x i8> %b) {
```

Comment:
Can this re-use the tests in llvm/test/CodeGen/AArch64/neon-extadd.ll, with GISel coverage?